raidnearby.py

```diff
@@ -696,6 +694,13 @@ def processRaidImage(self, raidfilename):
                         if update_raid == True:
 
                             try:
+                                if end_time < unix_time and self.config.USE_RAIDBOSS_TIMER:
+                                    end_time_temp = self.getEndTime(file_update_time, time_text)
+                                    if end_time_temp != -1:
+                                        end_time = end_time_temp
+                                        hatch_time = end_time - 6300
```

Comment:
Should that not be 2700 instead of 6300?
The db routine works from "hatch time"

Reply:
Correct. It’s hatch_time, so 2700.

Reply:
Correct. It’s hatch_time, so 2700.